resources/zib/zib-Encounter.xml

```diff
@@ -0,0 +1,237 @@
+    <name value="zib Encounter-v4.0.1(2020EN)" />
+  </mapping>
+  <mapping>
+    <identity value="zib-healthprofessional-v3.5-2020EN" />
+    <uri value="https://zibs.nl/wiki/HealthProfessional-v3.5(2020EN)" />
+    <name value="zib HealthProfessional-v3.5(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Encounter" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Encounter" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Encounter">
+      <path value="Encounter" />
+      <short value="Encounter" />
+      <comment value="Start date and end date can be aligned if only one date is known." />
+      <alias value="Contact" />
+      <mapping>
+        <identity value="zib-encounter-v4.0.1-2020EN" />
+        <map value="NL-CM:15.1.1" />
+        <comment value="Encounter" />
+      </mapping>
+    </element>
+    <element id="Encounter.class">
+      <path value="Encounter.class" />
+      <short value="ContactType" />
+      <definition value="The type of contact." />
+      <alias value="ContactType" />
+      <binding>
+        <strength value="extensible" />
+        <description value="ContactTypeCodelijst" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.15.1.1--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-encounter-v4.0.1-2020EN" />
+        <map value="NL-CM:15.1.2" />
+        <comment value="ContactType" />
+      </mapping>
+    </element>
+    <element id="Encounter.participant.type">
+      <path value="Encounter.participant.type" />
+      <short value="HealthProfessionalRole" />
+      <definition value="The role the health professional fulfils in the healthcare process. For health professionals, this could be for example attender, referrer or performer." />
+      <alias value="ZorgverlenerRol" />
+      <mapping>
+        <identity value="zib-healthprofessional-v3.5-2020EN" />
+        <map value="NL-CM:17.1.5" />
+        <comment value="HealthProfessionalRole" />
+      </mapping>
+    </element>
+    <element id="Encounter.participant.individual">
+      <path value="Encounter.participant.individual" />
+      <short value="ContactWith" />
+      <definition value="The health professional with whom the contact took or will take place. The specialty and role of the health professional can be entered in the HealthProfessional information model." />
+      <alias value="ContactMet" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Practitioner" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/PractitionerRole" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/RelatedPerson" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-HealthProfessional-Practitioner" />
```

Comment:
We volgen hier nu een andere methodiek voor https://github.com/Nictiz/Nictiz-R4-zib2020/issues/41

Reply:
Changed reference to:  http://nictiz.nl/fhir/StructureDefinition/pattern-HealthProfessionalReference